lib/types.ts

```diff
@@ -16,12 +16,7 @@ export type HighlightItem = {
   // From providers
   range: Range
   class?: string
-  created: (parameters: {
-    marker: DisplayMarker
-    element: HTMLElement
```

Comment:
This changes the API of `HighlightItem.created` because Atom's built-in highlight class doesn't create a separate element for the decorations.

@steelbrain Is this something that we should stick to, or is it acceptable to remove it?